### PR TITLE
feat: update formula documentation to include new fields

### PR DESCRIPTION
**Pull request by JorTurFer**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Review by tomkerkhove — COMMENTED** [review]
Can you also update the "Scaling Modifiers (Experimental) " section please?

**Comment by JorTurFer** [comment]
> Can you also update the "Scaling Modifiers (Experimental) " section please?

sure!

File: content/docs/2.12/concepts/scaling-deployments.md [hunk 1]
```diff
@@ -286,17 +296,17 @@ advanced:
   scalingModifiers:
     formula: "(trig_one + trig_two)/2"
     target: "2"
+    activationTarget: "2"
```
Comment (zroubalik) [inline content/docs/2.12/concepts/scaling-deployments.md:299]: Could you please add activationTarget with an explanation to examples below?

Comment (JorTurFer) [reply content/docs/2.12/concepts/scaling-deployments.md:299]: done

**Review by zroubalik — APPROVED** [review]
LGTM

**Pull request merged**